aidial_adapter_bedrock/llm/converse/input.py

```diff
@@ -146,12 +170,31 @@ def tool_result_to_content_part(
         }
 
 
-def to_converse_image_type(type: str) -> str:
-    if type == "image/png":
-        return "png"
-    if type == "image/jpeg":
-        return "jpeg"
-    raise RuntimeServerError(f"Unsupported image type: {type}")
+def to_converse_multi_modal_part(
+    resource: Resource,
+) -> ConverseImagePart | ConverseDocumentPart:
+    if resource.type in IMAGE_MIME_TO_CONVERSE_TYPE:
+        return ConverseImagePart(
+            image=ConverseImagePartConfig(
+                format=IMAGE_MIME_TO_CONVERSE_TYPE[resource.type],
+                source={"bytes": resource.data},
+            )
+        )
+    elif resource.type in DOCUMENT_MIME_TO_CONVERSE_TYPE:
+        return ConverseDocumentPart(
+            document=ConverseDocumentPartConfig(
+                format=DOCUMENT_MIME_TO_CONVERSE_TYPE[resource.type],
+                # Passing generic name, since provider explicitly stated,
+                # that this field is vulnerable for prompt injection, because the model might
+                # inadvertently interpret it as instructions.
+                name=str(uuid.uuid4()),
```

Comment:
It leads to a strange UX - the user attaches a list of files with different names and asks which one contains information about something they need.

And I already heard about use cases like this one but in the context of Gemini _(which doesn't allow to name files, which is unfortunate)_.

The model answers with uuid :(

I don't think that adapters should do any kind of **guardrails**.
Such logic should reside somewhere else e.g. in interceptors.

Reply:
Fixed, if user will pass attachment with `title`, I will pass it as a name

Also, introduced `sanitize_document_name` function, since Converse name doesn't allow a lot of characters (`some_file.txt` is invalid due to `.`)
So I need to do `some_file.txt -> some_file_txt`